include/sys/pathname.h

```diff
@@ -54,8 +54,10 @@ extern "C" {
  */
 typedef struct pathname {
 	char	*pn_buf;		/* underlying storage */
+#if 0 /* unused in ZoL */
```

Comment:
with efforts underway to unite the ZFSonLinux and FreeBSD source bases, perhaps it would be more future-proof if we can use a #define that specifies Linux instead of `0`?

Reply:
> we can use a #define that specifies Linux instead of 0?

If there is any good macro for it, I think so.

Regarding FreeBSD, FreeBSD-current doesn't use this struct to begin with. They have changed vnops side not to use `pathname_t` since BSD's don't have that struct just as Linux doesn't.

ZoL seems to have a policy to maintain Solaris VFS i/f where possible. ZoF is same as ZoL regarding this struct.

Reply:
thanks for the due dilligence